index.js

```diff
@@ -1,27 +1,23 @@
-
-
-function menu(){
-  let nav= document.getElementById("nav-header")  
-    nav.style.display="flex"
-    let icon= document.getElementById('close-icon')
-    let burger=document.getElementById('burger')
-    burger.style.display="none"
-    icon.style.display="inline-block"
-    document.getElementById('img-deco1').style.display="block"
-    document.getElementById('img-dots').style.display="block"
-    document.getElementById('img-deco2').style.display="block"
-
+function menu() {
+  const nav = document.getElementById('nav-header');
+  nav.style.display = 'flex';
+  const icon = document.getElementById('close-icon');
+  const burger = document.getElementById('burger');
+  burger.style.display = 'none';
+  icon.style.display = 'inline-block';
+  document.getElementById('img-deco1').style.display = 'block';
+  document.getElementById('img-dots').style.display = 'block';
+  document.getElementById('img-deco2').style.display = 'block';
 }
 
-function closeMenu(){
-    let nav= document.getElementById("nav-header")  
-    nav.style.display="none"
-    let icon= document.getElementById('close-icon')
-    let burger=document.getElementById('burger')
-    burger.style.display="flex"
-    icon.style.display="none"
-    document.getElementById('img-deco1').style.display="none"
-    document.getElementById('img-dots').style.display="none"
-    document.getElementById('img-deco2').style.display="none"
-
+function closeMenu() {
+  const nav = document.getElementById('nav-header');
+  nav.style.display = 'none';
+  const icon = document.getElementById('close-icon');
+  const burger = document.getElementById('burger');
+  burger.style.display = 'flex';
+  icon.style.display = 'none';
+  document.getElementById('img-deco1').style.display = 'none';
+  document.getElementById('img-dots').style.display = 'none';
+  document.getElementById('img-deco2').style.display = 'none';
```

Comment:
- [ ] As requested by the previous reviewer, when the user clicks (or taps) any mobile menu option, a correct part of the page is displayed and the mobile menu should close. currently, it's displaying the right path, but clicking any mobile menu option is not closing the mobile menu because when you scroll back up, it is still visible. Kindly add this feature.